source/package.json

```diff
@@ -19,14 +19,15 @@
   },
   "cicd": "false",
   "isROMode": "false",
-  "enableKendra": "false",
+  "enableComprehendMedical": "false",
+  "enableKendra": "true",
```

Comment:
Is kendra now going to be enabled by default?

Reply:
nope my bad.